confignode/src/main/java/org/apache/iotdb/confignode/manager/load/LoadManager.java

```diff
@@ -69,8 +73,8 @@ public class LoadManager {
 
   private final long heartbeatInterval =
       ConfigNodeDescriptor.getInstance().getConf().getHeartbeatInterval();
-  // Map<NodeId, HeartbeatCache>
-  private final Map<Integer, HeartbeatCache> heartbeatCacheMap;
+  // Map<NodeId, DataNodeHeartbeatCache>
```

Comment:
heartbeatCacheMap has DataNodeHeartbeatCache and ConfigNodeHeartbeatCache  together?

Reply:
yes

Reply:
I'll change the notes here